qutebrowser/commands/cmdutils.py

```diff
@@ -31,6 +31,22 @@
 cmd_dict = {}
 
 
+def quote(s):
```

Comment:
You'll need to adjust failing tests in `tests/unit/completion/test_completer.py` for this move. Also, can you please add some tests for this to `tests/unit/commands/test_cmdutils.py` as `cmdutils.py` is one of the files where tests are enforced?